gulpfile.js

```diff
@@ -293,14 +292,16 @@ gulp.task('template:elevation', () => {
 });
 
 gulp.task('template:icons', () => {
-  gulp
-    .src('node_modules/bpk-svgs/dist/svgs/icons/**/*.svg')
+  const src = 'node_modules/bpk-svgs/dist/svgs/icons/**/*.svg';
+  return gulp
+    .src(src)
     .pipe(through.obj(convertToXml))
     .pipe(gulp.dest(PATHS.drawableRes));
 });
 
-gulp.task('default', () => {
-  runSequence(
+gulp.task(
+  'default',
+  gulp.series(
```

Comment:
Was `gulp.series` added in v 4? Same behaviour as the `runSequence` plugin?

Reply:
yes